ui/src/app/views/workspace/component.ts

```diff
@@ -140,6 +141,19 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
     this.wsId = wsid;
   }
 
+  private loadUserRoles() {
+    workspacesApi().getWorkspaceUserRoles(this.wsNamespace, this.wsId)
+      .then(
+        resp => {
+          this.workspaceUserRoles = fp.sortBy('familyName', resp.items);
```

Comment:
We no longer need to sort here, right? Because we're sorting on the server-side?